src/services/db/GitFileSystemService.ts

```diff
@@ -341,7 +333,10 @@ export default class GitFileSystemService {
   }
 
   // Push the latest changes to upstream Git hosting provider
-  push(repoName: string): ResultAsync<string, GitFileSystemError> {
+  push(
+    repoName: string,
+    isForce?: boolean
```

Comment:
maybe can put as default value isForce = false

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/895/commits/5e49f77a80c941a1dac4e3dd3fc857e79f3ef878!